src/app/index.run.coffee

```diff
@@ -44,12 +44,19 @@ angular.module 'mnoEnterpriseAngular'
 
     $translate.use(locale)
   )
-
-   # Display flash messages from the backend in toastr
-   # They're passed this way:
-   #    ?flash={"msg":"An error message.","type":"error"}
+  # Display flash messages from the backend in toastr
+  # They're passed this way:
+  #    ?flash={"msg":"An error message.","type":"error"}
```

Comment:
@ouranos I am not quite sure what yo mean by removing the extra line, as the code is the same way it was added and I did not change it, see in the following PR, https://github.com/maestrano/mno-enterprise-angular/commit/bc44fc7a42a5d3f9f95b68fa27e3a8e2e00abee0

Check it the change I did worked for you please.